.circleci/config.yml

```diff
@@ -474,6 +474,9 @@ jobs:
             cd $HOME/emsdk
             git reset --hard
             git pull
+            # FIXME(ilammy, 2020-07-07): unpin emsdk version [T1698]
```

Comment:
amount of `FIXME` tags for emsdk is sad :(

Reply:
Expect a new one on August 7 😂

Reply:
😭